### PR TITLE
[FEATURE] Add new expectation expect_sum_of_columns_to_be_equal (pandas)

### PR DESCRIPTION
**Pull request by chipmyersjr**
Please annotate your PR title to describe what the PR does, then give a brief bulleted description of your PR below. PR titles should begin with [BUGFIX], [ENHANCEMENT], [FEATURE],  [DOCS], or [MAINTENANCE]. If a new feature introduces breaking changes for the Great Expectations API or configuration files, please also add [BREAKING]. You can read about the tags in our [contributor checklist](https://docs.greatexpectations.io/en/latest/contributing/contribution_checklist.html).

Changes proposed in this pull request:
- Add new expectation expect_sum_of_columns_to_be_equal for pandas backend. issue #1452
- 
-

After submitting your PR, CI checks will run and @tiny-tim-bot will check for your CLA signature.

For a PR with nontrivial changes, we review with both design-centric and code-centric lenses.

In a design review, we aim to ensure that the PR is consistent with our relationship to the open source community, with our software architecture and abstractions, and with our users' needs and expectations. That review often starts well before a PR, for example in github issues or slack, so please link to relevant conversations in notes below to help reviewers understand and approve your PR more quickly (e.g. `closes #123`).

Previous Design Review notes:
-
-
-

Thank you for submitting!

### CODE REVIEW

**Comment by chipmyersjr** [comment]
@Shinnnyshinshin Requested changes made

**Comment by chipmyersjr** [comment]
continued from https://github.com/great-expectations/great_expectations/pull/1664

File: docs/changelog.rst [hunk 1]
```diff
@@ -7,7 +7,7 @@ Changelog
 Develop
 -----------------
 * [BUGFIX] SuiteEditNotebookRenderer no longer break GCS and S3 data paths
-
+* [FEATURE] Add new exception: expect_multicolumn_sum_to_equal for `pandas` and `Spark` (thanks @chipmyersjr)!
```
Comment (Shinnnyshinshin) [inline docs/changelog.rst:10]: added change description to `Develop` section, along with thanks to @chipmyersjr

**Review by Shinnnyshinshin — APPROVED** [review]
Thank you very much @chipmyersjr  :) Looks great to me

**Pull request merged**